### PR TITLE
Redesign form section of mobile version

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Changes
* [x] Added a contact form with 4 inputs, 1 textarea, and 1 button.
* [X] Added styles to the form, making it responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,10 +220,12 @@ <h2>
...
         <form action="https://formspree.io/f/mleznykr" method="POST" class="form">
             <div class="form_list">
-                <input class="form_input form_name form_firs_name" type="text" placeholder="First name" name="user_name"
+                <input class="form_input form_name form_firs_name desktop" type="text" placeholder="First name" name="user_name"
                     required maxlength="30" />
-                <input class="form_input form_name form_last_name" type="text" placeholder="Last name"
+                <input class="form_input form_name form_last_name desktop" type="text" placeholder="Last name"
                     name="user_last_name" required maxlength="30" />
+                <input class="form_input form_name form_full_name mobile" type="text" placeholder="Full name"
+                    name="user_last_name" required maxlength="60" />
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly follow the figma design and add the social media icons below the contact form.
- [ ] Kindly make sure to align the text area with the name and email address by increasing its width.
- Your design:
Screenshot 2022-03-30 at 12 43 14 AM

- Figma design:
Screenshot 2022-03-30 at 12 43 01 AM

File: styles.css [hunk 2]
```diff
@@ -437,22 +447,20 @@ font-family: 'Roboto', sans-serif;
...
-  border: none;
-  padding: 10px 15px;
-  transition: all 400ms;
-}
-
 .form_btn button:hover {
   background-color: #fff5e1;
   border: 1px solid #f55800;
   color: #f55800;
 }
```
Comment (sja-thedude) [inline styles.css:458]: - [ ] Kindly use flexbox to layout the elements for the contact form.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @lucabenaventew, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Linters passing
✔️ Good job working so far

## Required 
- [ ] Kindly update your readme with a good description and make sure you add the collaborator info.
- [ ] Kindly add a descriptive PR description.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lucabenaventew** [comment]
Thank You @sja-thedude for your review, all requirements have been completed.

**Comment by roshan-bajgain** [comment]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight:
- 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error at present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Optional suggestions:
-
- [ ]  [OPTIONAL] It would be better if you can add a live demo link for your project in your readme file.  

_I am going to approve your project but hope so you will add the live demo link._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**